chat-demo-android/src/main/AndroidManifest.xml

```diff
@@ -68,9 +68,10 @@
 
         <service
             android:name=".services.FCMInstanceIDService"
-            android:exported="true">
+            android:exported="false">
```

Comment:
not related to crashlytics also, split off

Reply:
done